### ISSUE TITLE
NVDA have an problem on windows server 2012 r2's server manager

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5420]
**Reported by ichenyao2000 on 2015-10-15 08:43**
I have an based-on Windows Server 2012 R2 learning machine. when I trying installing roles and features by server manager, but the NVDA cannot be read role and feature's install-state, for example: installed, has not been installed. so, i can only using windows narrator, i would like NVDA adding support for Windows Server 2012 R2's Server Manager.

**Comment by leonardder** [comment]
Last time I tried, it looked a bit quirky but I recall I was able to see the state of roles correctly. Not surethough.

Duplicate of #967, though that mentions server 2008. However, since #967 is closed, lets leave this open :)

**Comment by Adriani90** [comment]
@ichenyao2000 are you still having this issue in Windows server 2012 R2 with NVDA 2023.2?

**Comment by Adriani90** [comment]
Ok it seems the authors are no longer available on Github and we didn't get any updates from the initial author since many years. i am closing as abandoned.
